blog/docs/articles/kubevirt_meets_eventing.md

```diff
@@ -267,7 +270,7 @@ Password for user postgres:
 
 The business-logic in this example is written in Python. It'll process the incoming tailored event, establish a connection to the PostgreSQL DB and will write the desired data into the columns of the DB `vmdb`. Keep in mind, that you have the freedom of deciding in which programming language you write your business-logic.
 
-The code for the used function can be found on Github here: [KubeVirt PostgreSQL Knative Function Example](https://github.com/rguske/knative-functions/tree/main/kn-py-vmdata-psql-fn)
+The code for the used function can be found on Github here: [KubeVirt PostgreSQL Knative Function Example](https://github.com/rguske/knative-functions/tree/main/kn-py-vmdata-psql-fn){:target="_blank"}
```

Comment:
unfortunately this is outdated python runtime from knative. our docs are lacking. see: https://github.com/knative/func/issues/2891

Reply:
I see! But this should not prevent the blog post from being published, right? I can fix this later.